ts/src/kucoin.ts

```diff
@@ -967,7 +969,7 @@ export default class kucoin extends Exchange {
                     'name': this.safeString (chain, 'chainName'),
                     'code': networkCode,
                     'active': isWithdrawalEnabled && isDepositEnabled,
-                    'fee': this.safeNumber (chain, 'withdrawalMinFee'),
+                    'fee': this.safeNumber (chain, 'withdrawMaxFee', this.safeNumber (chain, 'withdrawalMinFee')),
```

Comment:
Could you explain the reason of this change?
<img width="746" alt="image" src="https://github.com/cedelabs/ccxt/assets/57397713/56763ae8-1ca3-4661-9601-f79fe820cc6a">

Reply:
In case there is max and min fees i think it's more appropriate to display the maximum fees